koala/cogs/insights/cog.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+"""
+Koala Bot Insights Cog Code
+"""
+# Futures
+
+# Built-in/Generic Imports
+
+# Libs
+from discord.ext import commands
+
+# Own modules
+import koalabot
+from .log import logger
+
+# Constants
+
+# Variables
+
+
+class Insights(commands.Cog, name="Insights"):
+    """
+    A discord.py cog with commands to give insight into information about the servers the bot is in
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+        
+    @commands.command(name="insights")
+    @commands.check(koalabot.is_owner)
+    async def insights(self, ctx):
+        """
+        Lists the number of servers the bot is in, and the total number of members across all of those servers
+        (includes double counting)
+        :param ctx: Context of the command
+        """
+
+        message = (f"Insights:\nThis bot is in a total of {len(self.bot.guilds)} servers.\nThere are a total "
+                   f"of {sum([len(guild.members) for guild in self.bot.guilds])} members across these servers.")
```

Comment:
Should follow consistent approach for multiple lines (+ instead of brackets).
 Also need to move to core.py